.github/workflows/generateAndBuildAllTemplates.sh

```diff
@@ -0,0 +1,12 @@
+npm install -g yo
```

Comment:
The name of this file should be improved for consistency (we usually use `-`).
Example: `frontend-application.ts`, `ci-cd.yaml`.